digitalpublicgoods/santempi.json

```diff
@@ -0,0 +1,81 @@
+    ],
+    "evidenceStandardSupport": [
+      "https://github.com/santedb/hl7-testing-tool"
+    ],
+    "bestPracticesList": [
+      "SantéSuite software development processes and team follow the Principles for Digital Development, we work side-by-side with the country Ministry of Health frontline workers, digital health experts, management, as well as key implementation partners to foster “ground truth” evidence-based improvement and inspire and drive our innovation strategy. SanteSuite uses a variety of best practices and processes depending on client needs and capacity to participate. For complex projects we follow a Disciplined Agile Delivery Approach which combines the rigor of traditional processes, such as Waterfall and Unified Processes, with the responsiveness of Agile. At a minimum, we leverage Kanban and Scrum (Scrumban) for delivery of smaller projects where our clients do not have the capacity to participate in a full agile delivery. We implement a quality assurance plan after each iteration ensuring that all modules and features work as they are expected. We follow the test-driven development methodology, which ensure that all components or modules are thoroughly tested before they are incorporated into the final solution. Accompanying documentation is provided with each release, including technical documentation and release notes / change logs."
+    ]
+  },
+  "doNoHarm": {
+    "preventHarm": {
+      "stepsToPreventHarm": "Yes",
+      "additionalInfoMechanismProcessesPolicies": "SantéSuite solutions including SantéMPI all employ a “Privacy by Design” philosophy and software development lifecycle. Protecting privacy is an integral part of implementing a quality assurance plan after each development iteration ensuring that all modules and features work as they are expected. We follow the test-driven development methodology, which ensure that all components or modules are thoroughly tested for privacy compliance before they are incorporated into the final solution."
+    },
+    "dataPrivacySecurity": {
+      "collectsPII": "Yes",
+      "typesOfDataCollected": [
+        "name, date of birth, gender, birthplace, address, contact information such as phone number, health identification number(s), birth and death registration information"
+      ],
+      "thirdPartyDataSharing": "No",
+      "ensurePrivacySecurity": "Yes"
+    },
+    "inappropriateIllegalContent": {
+      "collectStoreDistribute": "No"
+    },
+    "protectionFromHarassment": {
+      "userInteraction": "Yes",
+      "addressSafetySecurityUnderageUsers": "Yes",
+      "stepsAddressRiskPreventSafetyUnderageUsers": [
+        "Currently primarily via Slack and Github and limited to technical software developers. SantéSuite does not operate an open online discussion forum."
+      ],
+      "griefAbuseHarassmentProtection": "Yes",
+      "harassmentProtectionSteps": [
+        "Not applicable"
```

Comment:
Why is this marked `Not Applicable` if the answer to the previous question is `Yes` ?

Reply:
Resolved in commit 60173d8.